groups/sig-release/groups.yaml

```diff
@@ -360,6 +360,7 @@ groups:
       - sreeramvenkitesh@gmail.com # v1.34 Release Lead Shadow
       - wendyha.sut@gmail.com # v1.34 Release Lead Shadow
     members:
+      - admin@mb-consulting.dev # v1.34 Release Branch Manager
```

Comment:
who is `admin@mb-consulting.dev` ?

Should we allow this kind of email?

cc @jeremyrickard @justaugustus @saschagrunert

Reply:
It's @mbianchidev